great_expectations/rule_based_profiler/types/data_assistant_result/volume_data_assistant_result.py

```diff
@@ -41,29 +41,38 @@ def plot(
                 "You may either use `include_column_names` or `exclude_column_names` (but not both)."
             )
 
-        charts: List[alt.Chart] = []
+        display_charts: List[alt.Chart] = []
+        return_charts: List[alt.Chart] = []
```

Comment:
Could we update the docstring to reflect the difference between these two?

Reply:
✅